test-definitions/fern/apis/exhaustive/definition/endpoints/params.yml

```diff
@@ -10,6 +10,16 @@ service:
       method: GET
       response: string
 
+    getWithInlinePath:
+      docs: GET with path param
+      path: /path/{param}
+      method: GET
+      response: string
+      request:
+        name: getWithInlinePath
+        path-parameters:
+          param: string
```

Comment:
We might actually want to put this in the `path-parameters` test definition alongside the rest ([link](https://github.com/fern-api/fern/blob/9e2fa9c277c17b90058ea449b5fb21dec88f0f83/test-definitions/fern/apis/path-parameters/definition/user.yml#L27)). I'm fine with it here if you prefer though (since this is supposed to capture _everything_.

Plus, if it's any easier, we might just be able to lean on `path-parameters` on its own so you don't need to update all these seed snaphots across every language.